sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -674,3 +678,35 @@ def test_lasso_non_float_y():
         clf_float = model(fit_intercept=False)
         clf_float.fit(X, y_float)
         assert_array_equal(clf.coef_, clf_float.coef_)
+
+
+def test_lasso_cv_best_score():
+    X, y, X_test, y_test = build_dataset(n_features=20)
+    grid_cv = GridSearchCV(
+        Lasso(), {'alpha': [0.1, 1., 10.]}, scoring='mean_squared_error',
+        cv=KFold(3)
+    ).fit(X, y)
+    lasso_cv = LassoCV(alphas=[0.1, 1., 10.], cv=KFold(3)).fit(X, y)
+    assert_equal(lasso_cv.best_score_, -1 * grid_cv.best_score_)
```

Comment:
> I was surprised at you telling me to be more pragmatic. I think usually it is
> the other way around ;)

I was being stupid; I apologize. I am doing budgets right now. It's not
helping. Will be around soon at the sprint.